app/spdb/build.sbt

```diff
@@ -224,6 +226,37 @@ def swalker(version: Version) = AppConfig(
   )
 ) extending List(with_gogo(version), swalker_credentials(version))
 
+// DNAVARRO
+def dnavarro(version: Version) = AppConfig(
```

Comment:
I don't get why is this happening, @jdnavarro could you check if your config is correct in `release` and `develop`?